linkml_model/model/schema/meta.yaml

```diff
@@ -1333,6 +1360,15 @@ classes:
     close_mappings:
       - qb:HierarchicalCodeList
 
+  alias_definition:
+    description: >-
+      object that contains meta data about a synonym or alias including where it came from (source)
+      and its scope (narrow, broad, etc.)
+    slots:
+      - alias_name
+      - alias_source
```

Comment:
Just added `source` #62 -- could be reused?

Reply:
moved to use 'source' instead of 'alias_source.'  I'm a little worried that we won't have uriorcurie for every alias source (eg.: I have sources in the biolink-model that are 'fake' curies to get around this requirement in the mapping objects like 'GAMMA:activator' where GAMMA is a made-up prefix), but, it could be that we change the alias source in this case to an actual source that might be broader than the made-up source.